deeplearning4j/pom.xml

```diff
@@ -452,6 +451,29 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>cuda</id>
+            <activation>
+                <property>
+                    <name>libnd4j.chip</name>
+                    <value>cuda</value>
+                </property>
+            </activation>
+            <modules>
+                <module>deeplearning4j-cuda</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>cuda2</id>
```

Comment:
That's just to work around a limitation of Maven profiles, nothing to do with CUDA version.

Reply:
Yeah, I know, but `cuda2` not really descriptive... That is why I though to add suffix to profile id...

Reply:
Ok, what about "libnd4j-cuda"? Not really more descriptive, they are both libnd4j, but one of them should be short like "cuda" because it makes it easier to use on the command line, and can be used without libnd4j anyway.

Reply:
Sounds good to me :+1: